pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/participant/OsteoParticipantService.java

```diff
@@ -103,6 +107,12 @@ protected boolean activityHasLaterVersion(List<Activities> activities, String ac
                 && !activity.getActivityVersion().equals("v1"));
```

Comment:
Migrated participants will have v1 too. 
Basically they will have v1 and also latest one v2 or v3 
OS1 only participants will have only v1

Reply:
Migrated meant OS1 uses who re-consented and moved to OS2/OS-pecgs

Reply:
Right. That is demonstrated by the tests added in a prior PR: `OsteoParticipantServiceTest.testCreateDefaultData` and `OsteoParticipantServiceTest.testCreateOsteo1DefaultData`. This is just adding that if there is no consent activities at all -- when a ptp has just registered -- DSM will add an OS2 tag to the osteo2 index, as described in the PR description. 
Since that is unclear, I will add a comment to the code.

Let me know if I missed your point.

Reply:
If this code is to deal with only PARTICIPANT_REGISTERED event, it should be fine because newly registered users will NOT have consent-v1.

Reply:
Will there be a participant registered event for the synthesized participants?
If not, then the OS1 branch will not be hit, and I will need to add the tag by hand.

Reply:
There will be participant registered event for the synthesized OS1 participants